src/Foundation/Commerce/code/Services/Account/AccountService.cs

```diff
@@ -0,0 +1,73 @@
+//    Copyright 2020 EPAM Systems, Inc.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+namespace Wooli.Foundation.Commerce.Services.Account
+{
+    using System;
+    using System.Collections.Generic;
+
+    using DependencyInjection;
+
+    using Models;
+    using Models.Account;
+    using Models.Entities.Addresses;
+
+    [Service(typeof(IAccountService), Lifetime = Lifetime.Singleton)]
+    public class AccountService : IAccountService
+    {
+        public Result<IEnumerable<Address>> AddAddress(string userName, Address address)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Result<ChangePasswordResultModel> ChangePassword(string email, string newPassword, string oldPassword)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Result<CreateAccountResultModel> CreateAccount(
+            string email,
+            string firstName,
+            string lastName,
+            string password)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Result<IEnumerable<Address>> GetAddress(string userName)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Result<IEnumerable<Address>> RemoveAddress(string userName, Address address)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Result<CommerceUserModel> UpdateAccount(string contactId, string firstName, string lastName)
```

Comment:
What is CommerceUserModel? If it's entity it should be `User` according to guidelines

Reply:
Return types will be changed with account service refactoring